workflow-templates/release-on-milestone-closed.yml

```diff
@@ -0,0 +1,45 @@
+name: "Automatic Releases"
+
+on:
+  milestone:
+    types:
+      - "closed"
+
+jobs:
+  release:
+    name: "GIT tag, release & create merge-up PR"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+
+      - name: "Release"
+        uses: "laminas/automatic-releases@1.0.1"
+        with:
+          command-name: "laminas:automatic-releases:release"
+        env:
+          "GITHUB_TOKEN": ${{ secrets.GITHUB_TOKEN }}
+          "SIGNING_SECRET_KEY": ${{ secrets.SIGNING_SECRET_KEY }}
+          "GIT_AUTHOR_NAME": ${{ secrets.GIT_AUTHOR_NAME }}
+          "GIT_AUTHOR_EMAIL": ${{ secrets.GIT_AUTHOR_EMAIL }}
+
+      - name: "Create Merge-Up Pull Request"
+        uses: "laminas/automatic-releases@1.0.1"
+        with:
+          command-name: "laminas:automatic-releases:create-merge-up-pull-request"
```

Comment:
This one will create a PR for an upmerge to the next higher branch. The branch names have to be named after a human logical name for a sorting that the laminas/automatic-releases is doing. Therefore no names like `master`. So my guess is that having mixed up name patterns like `2.x` and `2.0.x` are not advisable. At least for those starting with the default branch name and "up".

I don't know if there are still such cases out there, but there were a few renamings in the past (which broke the website build :smile:) so we need to review the current branch names of each repo that is going to use this just to be on the safe side.

Reply:
Note: this is now documented as per https://github.com/laminas/automatic-releases/pull/31

Reply:
@SenseException do we currently have branches named without `2.x` (literal `x`) ? Or were you just referring to ORM `2.7`?

Reply:
It was a literal x back then.

Reply:
_back then_ ? Do you mean we no longer have such branches?

Reply:
> I don't know if there are still such cases out there ... we need to review the current branch names of each repo that is going to use this just to be on the safe side.

We might have no such branches anymore. That's what we need to find out.

Reply:
I tried `for word in [Dd]octrine*; do cd $word; echo $word; git branch -r --list 'origin/*'  ; cd .. ; done` And I think apart from 2.x on DoctrineFixturesBundle, we're good.

Reply:
That branch has been inactive for 3 years BTW, so I don't think we even need to rename it, do we?

Reply:
If that means it is an unsupported version branch, then yes. I don't know if this is relevant for the docs on Symfony though: https://symfony.com/doc/2.x/bundles/DoctrineFixturesBundle/index.html

Reply:
Anyway, is this a blocker? We can still perform merges up to master manually and make the rename from master to something else independent from this.

Reply:
With your proposal of up-merging to master this shouldn't be a blocker IMHO.